src/js/utils/MenuUtils.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import _ from 'lodash';
+
+export function getMenuItemsComponent(section, key, active = false) {
```

Comment:
To me, all of these utill functions in `MenuUtils.jsx` like:

- _getMenuItemsComponent_ 
- _getSubsectionComponent_ 
- _getSectionComponent_ 
- _getConfigurationSubsections_

They should not be utills but a simple function components with props:
- `<MenuItem />`
- `<MenuSection />`
- `<MenuSubsection />`
- `<MenuConfigurationSubsection />`

and they should be in components directory

Reply:
nice idea, I just wanted to use already existing code wherever it was possible and it was one of them, as Menu was done like that:
```
 { _.map(menuConfig, (section, key) => {
          if (section.href) {
            return getSectionComponent(section, key);
          }
          if (section.subsections) {
            return getSubsectionComponent(section, key);
          }
          if (section.menuItems) {
            return getMenuItemsComponent(section, key);
          }
          return null;
        })
```
and those functions were at the top of the Menu component, so I wanted at least to move them to a separate file, but I'll probably proceed with your suggestion